--- gcc-4.6.4/configure.old	2014-01-28 16:55:11.097852645 +0400
+++ gcc-4.6.4/configure	2014-01-28 16:53:02.541852039 +0400
@@ -3231,7 +3231,7 @@
     libgloss_dir=arm
     ;;
   arm-*-elf* | strongarm-*-elf* | xscale-*-elf* | arm*-*-eabi* )
-    noconfigdirs="$noconfigdirs target-libffi target-qthreads"
+    noconfigdirs="$noconfigdirs target-libstdc++-v3 target-libffi target-qthreads"
     libgloss_dir=arm
     ;;
   arm*-*-linux-gnueabi)
@@ -3495,6 +3495,9 @@
   powerpc-*-eabi* | powerpcle-*-eabi* | powerpc-*-rtems* )
     libgloss_dir=rs6000
     ;;
+  powerpc-*-elf)
+    noconfigdirs="$noconfigdirs target-libstdc++-v3 ${libgcj}"
+    ;;
   rs6000-*-lynxos*)
     noconfigdirs="$noconfigdirs target-newlib gprof ${libgcj}"
     ;;
@@ -3508,7 +3511,7 @@
     noconfigdirs="$noconfigdirs ld binutils gprof target-libgloss ${libgcj}"
     ;;
   microblaze*)
-    noconfigdirs="$noconfigdirs gprof target-libssp ${libgcj}"
+    noconfigdirs="$noconfigdirs gprof target-libstdc++-v3 target-libssp ${libgcj}"
     ;;
   mips*-sde-elf*)
     noconfigdirs="$noconfigdirs ${libgcj}"
@@ -3534,10 +3537,10 @@
     noconfigdirs="$noconfigdirs target-newlib target-libgloss"
     ;;
   mips*-*-*)
-    noconfigdirs="$noconfigdirs gprof ${libgcj}"
+    noconfigdirs="$noconfigdirs gprof target-libstdc++-v3 ${libgcj}"
     libgloss_dir=mips
     ;;
-  romp-*-*)
+  romp-*-*target-libstdc++-v3 )
     noconfigdirs="$noconfigdirs bfd binutils ld gas opcodes target-libgloss ${libgcj}"
     ;;
   sh-*-* | sh64-*-*)
@@ -3558,7 +3561,7 @@
     libgloss_dir=sparc
     ;;
   sparc-*-elf*)
-    noconfigdirs="$noconfigdirs ${libgcj}"
+    noconfigdirs="$noconfigdirs target-libstdc++-v3 ${libgcj}"
     ;;
   sparc64-*-elf*)
     noconfigdirs="$noconfigdirs ${libgcj}"
