--- binutils-2.25.orig/bfd/elf32-msp430.c
+++ binutils-2.25/bfd/elf32-msp430.c
@@ -2547,6 +2547,7 @@ msp430_elf_is_target_special_symbol (bfd *abfd, asymbol *sym)
 #define elf_backend_relocate_section         elf32_msp430_relocate_section
 #define elf_backend_check_relocs             elf32_msp430_check_relocs
 #define elf_backend_can_gc_sections          1
+#define elf_backend_rela_normal              1
 #define elf_backend_final_write_processing   bfd_elf_msp430_final_write_processing
 #define elf_backend_object_p		     elf32_msp430_object_p
 #define bfd_elf32_bfd_relax_section	     msp430_elf_relax_section
